recreate policies branch (#8)
AlekEagle/cumulonimbus-frontend: TOS.md

# Cumulonimbus Terms of Service

Last Updated: September 15th, 2023

> This document describes the usage of Cumulonimbus (which from here on out may be referred to as `We`, `Us`, and/or `the Service`) allowed by you. By using the Service you are agreeing to these terms. THIS SERVICE IS PROVIDED AS-IS WITH ABSOLUTELY NO WARRANTY IN ANY WAY, SHAPE, OR FORM. WE ARE NOT RESPONSIBLE FOR ANY LOSS OF DATA. FAILURE TO ABIDE BY THESE TERMS MAY RESULT IN YOUR ACCOUNT BEING DISABLED (BANNED) OR TERMINATED. ANY ACTION TAKEN AGAINST YOUR ACCOUNT WILL BE ACCOMPANIED WITH A REASON, WHETHER IT BE BY EMAIL OR ON THE SERVICE ITSELF.

Reviewer: ```suggestion
This document describes the usage of Cumulonimbus (which from here on out may be referred to as `We`, `Us`, and/or `the Service`) allowed by you. By using the Service you are agreeing to these terms. THIS SERVICE IS PROVIDED AS-IS WITH ABSOLUTELY NO WARRANTY IN ANY WAY, SHAPE, OR FORM. WE ARE NOT RESPONSIBLE FOR ANY LOSS OF DATA. FAILURE TO ABIDE BY THESE TERMS MAY RESULT IN YOUR ACCOUNT BEING DISABLED ("BANNED") OR TERMINATED. ANY ACTION TAKEN AGAINST YOUR ACCOUNT WILL BE ACCOMPANIED WITH A REASON, WHETHER IT BE BY EMAIL OR ON THE SERVICE ITSELF.
```

> If you do not agree to these terms then usage must be discontinued immediately.

Reviewer: ```suggestion
If you do not agree to these terms, usage of the Service must be discontinued immediately.
```

> We will alert you of these terms being updated both by changing the "Last Updated" date above and by prompting you on the Service. If We deem the change great enough you may receive an email concerning the change of the Terms of Service. While you will be notified of any changes to these terms We still strongly recommend that you periodically review these terms.

Reviewer: ```suggestion
We will alert you of these terms being updated both by changing the "Last Updated" date above and by prompting you on the Service. If We deem the change great enough, you may receive an email concerning the change of the Terms of Service. While you will be notified of any changes to these terms, We still strongly recommend that you periodically review these terms.
```

> The source code of the Service is provided not to be used to create a hostile clone (duplicates of the Service to mislead, deceive, or act in any other malicious manner). Using the source code of the Service to create a hostile clone will result in defensive action.

Reviewer: ```suggestion
The source code of the Service is provided not to be used to create a "hostile clone" (duplicates of the Service to mislead, deceive, or act in any other malicious manner). Using the source code of the Service to create a "hostile clone" as described above will result in defensive action.
```

## Acceptable Use

> Any and all content uploaded by you is, and will always be your property. the Service is not liable for any user-generated content. The upload of illegal, harmful, and/or immoral content as defined within your jurisdiction, the state laws of Alaska and Missouri, and federal US laws is not allowed and will be removed with valid reason being sent to you via email. Continued violations may result in your account being disabled or terminated. Account termination will result in your account being deleted and all user generated content being deleted. Violations resulting in an account being disabled can be appealed, while account termination is permanent and cannot be reversed. Abuse of the API is not allowed and can result in your account being disabled or terminated.

Reviewer: ```suggestion
Any and all content uploaded by you is, and will always be Your property. the Service is not liable for any user-generated content. The upload of illegal, harmful, and/or immoral content as defined within Your jurisdiction, the state laws of Alaska and Missouri, and U.S. Federal Law is not allowed and will be removed with valid reason being sent to you via email. Continued violations may result in your account being disabled ("banned") or terminated. Account termination will result in Your account being deleted, as well as all user-generated content contained within Your account. Violations resulting in an account being disabled ("banned") can be appealed, while account termination is permanent and cannot be reversed. Abuse of the API is not allowed and can result in Your account being disabled ("banned") or terminated.
```

## Digital Millennium Copyright Act (DMCA) Claims

> If a user is to upload content that violates DMCA a request for a takedown can be filed at dmca@alekeagle.com with a subject of "Cumulonimbus DMCA Claim". Abuse of a DMCA Claim is not tolerated and will go ignored.

Reviewer: ```suggestion
If a user is to upload content that violates DMCA, a request for a take-down can be filed at dmca@alekeagle.com with a subject of "Cumulonimbus DMCA Claim". Abuse of a DMCA Claim is not tolerated and will go ignored.
```

## Contact

> If you have any questions please contact Us at contact@alekeagle.com

Reviewer: ```suggestion
If You have any questions please contact Us at contact@alekeagle.com
```
